modules/3d/include/opencv2/3d/ptcloud.hpp

```diff
@@ -61,17 +61,17 @@ CV_EXPORTS void randomSampling(OutputArray sampled_pts, InputArray input_pts,
 
 /**
  * @brief Point cloud sampling by Farthest Point Sampling(FPS).
- *
- * FPS Algorithm:
- *   Input: Point cloud *C*, *sampled_pts_size*, *dist_lower_limit*
- *   Initialize: Set sampled point cloud S to the empty set
- *   Step:
- *     1. Randomly take a seed point from C and take it from C to S;
- *     2. Find a point in C that is the farthest away from S and take it from C to S;
- *       (The distance from point to set S is the smallest distance from point to all points in S)
- *     3. Repeat *step 2* until the farthest distance of the point in C from S
- *       is less than *dist_lower_limit*, or the size of S is equal to *sampled_pts_size*.
- *   Output: Sampled point cloud S
+ * \n\n
```

Comment:
How about using `<br>`, in some places that are not lists, line breaks will increase the readability of the document.

Reply:
In general we should avoid them.

Reply:
you can check generated documentation for this PR here: http://pullrequest.opencv.org/buildbot/export/pr/21095/docs/

Reply:
Understood.
The following updates were made this time.
1. Removed `\n` from `ptcloud.hpp` comment.
2. update the default value of the argument `arrangement_of_points` in the `_getMatFromInputArray` function in `ptcloud_utils.hpp` from `0` to `1`, since the latter is more commonly used (such arrangement is easier for SIMD acceleration).
3. removed two unused functions from `ptcloud_utils.hpp` (I will add their optimized versions later in another PR)

> you can check generated documentation for this PR here: 
> http://pullrequest.opencv.org/buildbot/export/pr/21095/docs/

I just configured the environment for doxygen document generation on my computer and it will work better afterwards.